### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by katlazarova**
I added all the code as discussed, however the sidebar is not behaving as expected.

### CODE REVIEW

File: theme/sass/components/_blog-sidebar-component.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.blog-sidebar-component {
+
+  @include breakpoint($md) {
+    width: 40%
```
Comment (ElXyfer) [inline theme/sass/components/_blog-sidebar-component.scss]: Missing semi colon here.

File: theme/sass/components/_card-component.scss [hunk 2]
```diff
@@ -98,6 +98,8 @@
         &--horizontal {
           @include breakpoint($md) {
             flex-direction: row;
+            width: 60%;
+            float: left
           }
```
Comment (ElXyfer) [inline theme/sass/components/_card-component.scss]: Width and float left should be on the card component container not on the card which is where you've put it here.

File: theme/sass/components/_blog-sidebar-component.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    float: right;
```
Comment (ElXyfer) [inline theme/sass/components/_blog-sidebar-component.scss]: This should be floating to the left.

File: theme/sass/components/_blog-sidebar-component.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.blog-sidebar-component {
```
Comment (ElXyfer) [inline theme/sass/components/_blog-sidebar-component.scss]: This file needs to be added into `app.scss`

File: blog.php [hunk 5]
```diff
@@ -1,9 +1,20 @@
 <?php include 'includes/header.php';?>
 <div class="card-component-container bottom-spacer">
-	<div class="row-container row-container--horizontal">
-	</div> <!-- End of row-container -->
+    <div class="row-container row-container--horizontal">
+    </div> <!-- End of row-container -->
 </div> <!-- End of card-component-container -->
+<div class="blog-sidebar-component">
```
Comment (ElXyfer) [inline blog.php]: Change this class name to be just `sidebar-container` also change the class name in the sass file.

File: theme/sass/components/_blog-sidebar-component.scss (hunk 1 above)
Comment (katlazarova) [reply theme/sass/components/_blog-sidebar-component.scss]: done

File: theme/sass/components/_card-component.scss (hunk 2 above)
Comment (katlazarova) [reply theme/sass/components/_card-component.scss]: done

File: theme/sass/components/_blog-sidebar-component.scss (hunk 3 above)
Comment (katlazarova) [reply theme/sass/components/_blog-sidebar-component.scss]: No, it should be right.

File: theme/sass/components/_blog-sidebar-component.scss (hunk 4 above)
Comment (katlazarova) [reply theme/sass/components/_blog-sidebar-component.scss]: done! So sorry about this.

File: blog.php (hunk 5 above)
Comment (katlazarova) [reply blog.php]: done.

File: blog.php [hunk 6]
```diff
@@ -1,9 +1,20 @@
 <?php include 'includes/header.php';?>
-<div class="card-component-container bottom-spacer">
-	<div class="row-container row-container--horizontal">
-	</div> <!-- End of row-container -->
+<div class="card-component-container bottom-spacer card-component-container--horizontal">
+    <div class="row-container row-container--horizontal">
+    </div> <!-- End of row-container -->
 </div> <!-- End of card-component-container -->
+<div class="sidebar-container">
```
Comment (ElXyfer) [inline blog.php]: also add the class `sidebar` here

Comment (katlazarova) [reply blog.php]: done

File: theme/sass/components/_blog-sidebar-component.scss (hunk 3 above)
Comment (ElXyfer) [reply theme/sass/components/_blog-sidebar-component.scss]: They both work the same way so thats fine.

**Pull request merged**